app/models/user.rb

```diff
@@ -4,6 +4,8 @@ class User < ApplicationRecord
   has_secure_password
   paginates_per MAX_ITEMS_PER_PAGE
   before_create :confirmation_token
+  before_save { self.email = email.downcase! }
+  before_save { self.name = name.downcase! }
```

Comment:
I think, `before_save { email.downcase! }` and `before_save { name.downcase! }` will be enough.

Reply:
@evrusha you are right.
Also it will be better to use before_validation callbacks here.

Reply:
@fooooxmr @evrusha Updated